src/fromager/packagesettings.py

```diff
@@ -1065,6 +1071,7 @@ def __init__(
         self._patches_dir = patches_dir
         self._max_jobs = max_jobs
         self._pbi_cache: dict[Package, PackageBuildInfo] = {}
+        self.pre_built_override: set[NormalizedName] = set()
```

Comment:
I think it would make more sense to have `_pre_built_override` on the WorkContext object and then have a small API to add / get pre_built overrides. WDYT?

Reply:
I agree. Fixing it.